docs/learning/ui/framework/index.md

```diff
@@ -1,6 +1,6 @@
 # The ui-framework package
 
-The ui-framework package contains application fragments for Login, Project, iModel and View selection, and configuration of the application UI including the Backstage, Frontstages, Zones, Widgets, etc.
+The ui-framework package contains application fragments for Project, iModel and View selection, and configuration of the application UI including the Backstage, Frontstages, Zones, Widgets, etc.
```

Comment:
@bsteinbk do we really need a Project/IModel selector in ui-framework? Im not even sure what this even looks like. We have the bentley/imodel-select-react package which probably does the same thing i think. That'll be deprecated in favor of the IModel and Project browser components coming from the itwin/admin-components-react repo i think. 

Is the View Selector here the IModelIndex page used by DR?

Reply:
@aruniverse I do not know the history of those components. @JoeZman added those years ago and outside of ui-test-app I am not sure if they are used. @NancyMcCallB  or @calebmshafer may know.

Reply:
We removed those components a while back but never updated the ui-framework description. It should be something like:

The ui-framework package contains components for specifying the application UI consisting of the 
Backstage, Frontstages, Tool Bars, Status Bars, Widgets and Panels.

Reply:
Just a heads up some of these components are still in ui-framework: https://github1s.com/imodeljs/imodeljs/blob/HEAD/ui/framework/src/ui-framework/pickers/ViewSelector.tsx

Reply:
Yes, ViewSelector is a component used by some applications like PlantSight. It is a useful component and is not a "fragment" like the others were. The project & iModel pickers were moved to ui-test-app.